package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "mdast-typographer",
-  "version": "0.2.0",
-  "description": "Mdast plugin to make your typography better.",
+  "name": "mdast-textr",
+  "version": "1.0.0",
```

Comment:
pull-requests cannot merge git tags and github releases, so just decrease it for now